pkg/cmd/attestation/verify/verify.go

```diff
@@ -261,7 +262,7 @@ func runVerify(opts *Options) error {
 		return nil
 	}
 
-	opts.Logger.Printf("%s was attested by:\n", artifact.DigestWithAlg())
+	opts.Logger.Printf("%s was attested by %s with the following attributes:\n\n", artifact.DigestWithAlg(), text.Pluralize(len(verified), "attestation"))
```

Comment:
"sha256:foobar was attested by N attestations" is a bit awkward eh?

what about,

- drop `DigestWithAlg`, since that info is displayed on the very first line and is redundant?

and then something like:

```suggestion
	opts.Logger.Printf("The following %d %s matched the policy criteria:\n\n", len(verified), text.Pluralize(len(verified), "attestation"))
```